### PR TITLE
Get headers

### PR DESCRIPTION
**Pull request by industriousparadigm**
Added functionality whereby the client can access headers passed during the browser's request.

Client's tapestry.config.js file may now include, for instance ```headers: ['location', 'connection-speed']``` , and these will be available in the _tapestry object that is generated.

### CODE REVIEW

File: src/server/handlers/dynamic.js [hunk 1]
```diff
@@ -38,11 +38,20 @@ export default ({ server, config }) => {
       } else {
         log.debug(`Skipping cache: ${chalk.green(cacheKey)}`)
       }
+      // Get headers and filter by client config
+      const headers = {}
+      config.headers &&
```
Comment (thomasdigby) [inline src/server/handlers/dynamic.js]: We _should_ probably check that it is an array as well, so `.map()` will be available. `Array.isArray()`?

Comment (industriousparadigm) [reply src/server/handlers/dynamic.js]: Good point, the user could be mischievous and pass a NaN 😾

**Comment by thomasdigby** [comment]
This is looking great, bump the version as a minor update and add to the changelog and we'll get it merged and published

File: src/server/handlers/dynamic.js [hunk 2]
```diff
@@ -38,11 +38,22 @@ export default ({ server, config }) => {
       } else {
         log.debug(`Skipping cache: ${chalk.green(cacheKey)}`)
       }
+      // Get headers and filter by client config
+      const headers = {}
+      const configHeaders = config.headers
+      const requestHeaders = request.headers
+      if (Array.isArray(configHeaders))
+        configHeaders.forEach(key => {
+          Object.keys(requestHeaders).includes(key)
```
Comment (thomasdigby) [inline src/server/handlers/dynamic.js]: `Object.keys(requestHeaders).includes(key)` Nice.

**Comment by thomasdigby** [comment]
Excellent work, I'm going to merge this and publish it shortly

**Pull request merged**